packages/earth-map/src/pages/not-found/component.tsx

```diff
@@ -17,23 +17,17 @@
   specific language governing permissions and limitations under the License.
 */
 
+import { compose } from 'lodash/fp';
 import React from 'react';
-import { replace } from 'redux-first-router';
 
 import { Button, ErrorTemplate } from '@marapp/earth-shared';
 
-const NotFound = ({ resetStore }) => {
+const NotFound = ({ returnToHome, resetStore }) => {
```

Comment:
You could compose them in the connected (index.tsx) component and only pass one prop in. This guarantees you can't use them separately in the component